docs/_layouts/component.html

```diff
@@ -8,15 +8,17 @@
         {% include sidebar.html %}
     </aside>
         {% include component.html %}
-        <div id="edit-page">
-            <a class="a-btn add-new" href="{{ site.baseurl }}/admin/#/collections/{{ page.collection_name }}/new" title="Add a new component page in Netlify CMS">
-                <span class="a-btn_text">Add new component page</span>
-                <span class="a-btn_icon">{% include icons/plus.svg %}</span>
-            </a>
+        <div class="cms-shortcut" id="edit-page">
             <a class="a-btn" href="{{ site.baseurl }}/admin/#/collections/{{ page.collection_name }}/entries/{{ page.title | slugify }}" title="Edit this page in Netlify CMS">
                 <span class="a-btn_text">Edit this page</span>
                 <span class="a-btn_icon">{% include icons/edit.svg %}</span>
             </a>
         </div>
+        <div class="cms-shortcut" id="add-page">
+            <a class="a-btn add-new" href="{{ site.baseurl }}/admin/#/collections/{{ page.collection_name }}/new" title="Add a new Design System page in Netlify CMS">
```

Comment:
Looks like `add-new` CSS was removed in this PR. Also `add-new` isn't BEM, should be id, utility (`u-`) or modifier on `a-btn`.

Reply:
BEMMMMMMMMMMMMM